ProjectSettings/EditorBuildSettings.asset

```diff
@@ -20,4 +20,7 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/_BForBoss/_LevelDesign/Scenes/CKJ_Weapon_Sandbox.unity
     guid: e09dd7ffe880d1b49939853d42d0020a
+  - enabled: 1
+    path: Assets/_BForBoss/_Core/Scenes/AdditiveDebugScene.unity
+    guid: d05c69bfc85df9741a9cac5511badaab
```

Comment:
I actually don't like this implementation cause it adds the scene for the Prod build, which is wasteful.

Initially wanted to manually add the scene in out BuildPipeline for Development builds via [DevBuildScript.cs](https://github.com/PerigonGames/BForBoss/blob/develop/Assets/Editor/DevBuildScript.cs). But I am not sure how to force a Dev build to try it out and make sure it works

Reply:
Any idea @m4rtint @mikeslgoh ?

Reply:
When you make a manual build in UNity, there's a checkbox that says 
`developer build`

Reply:
imo, it's not the worse thing to have it in EditorBuildSettings, seeing it should only be one debug Scene, but i understand the concern.